tests/sb-util.test.ts

```diff
@@ -0,0 +1,33 @@
+import { initialize, ScratchProject } from '../src/sb-util';
+import process from 'process';
+
+test('ScratchProject not initialized on invalid options', () => {
+	expect.assertions(2);
+	
+	// Empty options object
+	try {
+		initialize({});
+	} catch (e) {
+		expect(e.message).toContain('Please provide one of the following options');
+	}
+
+	// Multiple options object
+	try {
+		initialize({ 'file' : 'foo.sb3', 'cloudId': 12345 });
+	} catch (e) {
+		expect(e.message).toContain('Multiple options found.');
+	}
+});
```

Comment:
These two try/catch blocks should be their own tests, ie. 

```js
test('ScratchProject not initialized on empty options', () => {
  // empty object test
});

test('ScratchProject not initialized on multiple present options', () => {
  // multiple present options
});
```

Should also include: 

```js
test('ScratchProject not initialized when no options object present', () => {
  // ...
});

test('ScratchProject not initialized when only invalid options present', () => {
  // ...
});
```

Reply:
> test('ScratchProject not initialized when no options object present', () => {
  // ...
});

> test('ScratchProject not initialized when only invalid options present', () => {
  // ...
});

No way to make these tests because TypeScript enforces that the initialize function must take an object of type ScratchProjectOptions. This is the type checking we get for free.